### PR TITLE
Adding contributing instructions

### LINKED ISSUE #516: Empty CONTRIBUTING.md
https://github.com/teamdigitale/dati-semantic-frontend/blob/main/CONTRIBUTING.md is empty, even if in the pull requests template there's an item for contributors to check:  

image

It should contain the guidelines to contribute to this repo.

### PR DESCRIPTION
**Pull request by AgostinoPurificato**
Added contibuting instructions extracted from the contributing.md file of "_dati-semantic-backend_" repository

## Description

This PR does: adding the contributing instructions extracted from the "_dati-semantic-backend_" repository.

Fixes #516 

@Clou-dia @mauropratesi

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+### Creating a new issue
+
+There are three templates for new issues: `Bug report`, `Feature request`
+and `General issue`, pick the relevant one and follow the instructions
+in that template.
+
+If you pick `General issue` please provide a good amount of detail in
+order to let other people clearly understand the issue.
+After the creation of the issue, the maintainers team will promptly triage
+it by assigning the proper labels.
```
Comment (bfabio) [inline CONTRIBUTING.md:27]: We don't have template issues on this repo, we should rewrite or remove this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,82 @@
...
+
+> :information_source: Please ensure that there is a pertinent issue related to
+> what you are proposing and also make sure that someone has already reviewed it
+> before proceeding
+
+1. [Fork the project](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
+
+2. Follow our [conventions regarding commits](#commit-messages) for your commit message
+
+3. Open a Pull request against `master`.
```
Comment (bfabio) [inline CONTRIBUTING.md:42]:
```suggestion
3. Open a Pull request against `main`.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1 +1,82 @@
...
+
+This repository adopts a simplified branch management system as follows:
+
+* `master` is stable and gets deployed automatically. Never push directly to it;
+* feature or fix branches are derived directly from master.
+
+### Releases
+
+TBA :)
+
```
Comment (bfabio) [inline CONTRIBUTING.md:79]: I don't think this applies to this repo: `main` is not deployed automatically. I think we can remove this part.

**Review by bfabio — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (AgostinoPurificato) [reply CONTRIBUTING.md]: Removed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (AgostinoPurificato) [reply CONTRIBUTING.md]: Removed.

**Review by bfabio — APPROVED** [review]

**Comment by bfabio** [comment]
We should probably backport the same changes to `dati-semantic-backend`'s CONTRIBUTING.md, if that is where it came from.

**Comment by AgostinoPurificato** [comment]
@bfabio it seems like I cannot merge the "main" with this branch because of the failed checks....

**Pull request merged**